justfile

```diff
@@ -5,3 +5,7 @@
 install:
     mkdir -p ~/.local/bin
     ln -sf $(pwd)/bin/* ~/.local/bin
+
+# Re-build the README Markdown file.
+build-readme:
+    quarto render README.qmd
```

Comment:
Also curious about this :) 
I have seem many repos with both a `md` and a `qmd` version of the `README`. 
Do you mind elaborating on why we want both? And why only in this repo?

Reply:
The Quarto `qmd` file contains code that needs to be executed. Running Quarto to convert it from `qmd` to `md` will also execute the code (in this case, bash/shell) and insert the output into the document. Just like how my R courses as well as the Sprout Python usage code guide does it. So the `md` file is only the generated output of the `qmd` file.